components/TrackingComplaint/Form/StepFour.vue

```diff
@@ -8,7 +8,10 @@
         tag="section"
         name="Kota/Kabupaten"
       >
-        <label for="city" class="font-roboto font-medium text-black text-[14px] leading-[20px]">
+        <label
+          for="city"
+          class="font-roboto font-medium text-black text-[14px] leading-[20px]"
```

Comment:
`text-sm` for life
<img width="288" alt="Screenshot 2025-03-04 at 14 17 40" src="https://github.com/user-attachments/assets/e5a40268-e748-493c-8b79-4422bcb3b20c" />

Reply:
fix next PR sorry , selanjut nya ngerjain bagian yg ini,

gak sengaja ke prettier bagian yg ini wkwk